src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,77 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+type Props = {
+  PrepearedItems: string[];
+  Pages: number;
+  CurrentPage: number;
+  setCurrentPage: (value: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  PrepearedItems,
+  Pages,
+  CurrentPage,
+  setCurrentPage,
+}: Props) => {
+  const pages = Array.from({ length: Pages }, (_, index) => index + 1);
+
+  return (
+    <>
+      <ul className="pagination">
+        <li
+          className={cn('page-item', { disabled: CurrentPage === pages[0] })}
+          onClick={() =>
+            CurrentPage > pages[0] && setCurrentPage(CurrentPage - 1)
+          }
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={CurrentPage === pages[0]}
+          >
+            «
+          </a>
+        </li>
+
+        {pages.map(page => (
+          <li
+            key={page}
+            className={`page-item ${cn({ active: CurrentPage === page })}`}
+            onClick={() => setCurrentPage(page)}
+          >
+            <a data-cy="pageLink" className="page-link" href="#1">
```

Comment:
set current page at `href`
```suggestion
            <a data-cy="pageLink" className="page-link" href="#1">
```